systemverilog-plugin/UhdmAst.cc

```diff
@@ -2574,6 +2585,17 @@ void UhdmAst::process_enum_typespec()
     if (range) {
         delete range;
     }
+    if (current_node->str.empty()) {
+        // anonymous typespec
+        std::string typedef_name = "$systemverilog_plugin$anonymous_enum" + std::to_string(shared.next_anonymous_enum_typedef_id());
+        current_node->str = typedef_name;
+        auto current_scope = find_ancestor({AST::AST_PACKAGE, AST::AST_MODULE, AST::AST_BLOCK, AST::AST_GENBLOCK});
+        uhdmast_assert(current_scope != nullptr);
+        move_type_to_new_typedef(current_scope, current_node);
+        current_node = new AST::AstNode(AST::AST_WIRETYPE);
```

Comment:
```suggestion
        current_node = make_node(AST::AST_WIRETYPE);
```
It sets file location info from obj_h which is nice when debugging.